R/makeTreeSummarizedExperimentFromBiom.R

```diff
@@ -224,3 +244,77 @@ makeTreeSummarizedExperimentFromBiom <- function(obj, ...){
     }
     return(colname)    
 }
+
+# Detect and clean non wanted characters from Taxonomy data if needed.
+.detect_taxa_artifacts_and_clean <- function(x, patterns) {
+    
+    # No cleaning
+    if (is.null(patterns)) {
+        warning("No Taxonomy cleaning was done to the data.",
+                call. = FALSE)
+    }
+    # Automatic cleaning
+    else if (patterns=="auto") {
+        # General regex pattern
+        PATTERN <- "[[:alnum:]]|-|_|\\[|\\]|,|;\\||[[:space:]]"
+        patterns <- .detect_taxa_artifacts(x, PATTERN, invert=TRUE)
+        # No artifacts found
+        if (patterns=="") {
+            warning("No artifacts found in the Taxonomy data.",
+                    call. = FALSE)
```

Comment:
This should be a "message" rather than a warning. 

However I would perhaps have a preference for just no extra messages when everything goes well here and there are no artifacts.

Reply:
ok I will remove it.

Reply:
I would only throw a warning if something is modified, otherwise none.